crates/rust-analyzer/src/global_state.rs

```diff
@@ -361,6 +365,7 @@ impl GlobalState {
             (change, modified_rust_files, workspace_structure_change)
         };
 
+        let _p = span!(Level::INFO, "GlobalState::process_changes/apply_change");
```

Comment:
`.entered()`?

Reply:
ugh, someone should make a lint for this...